src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeHighRiskCellConfigurator.swift

```diff
@@ -96,4 +96,24 @@ final class HomeHighRiskCellConfigurator: HomeRiskLevelCellConfigurator {
 
 		setupAccessibility(cell)
 	}
+
+	// MARK: Hashable
+
+	override func hash(into hasher: inout Swift.Hasher) {
+		super.hash(into: &hasher)
+		hasher.combine(numberRiskContacts)
+		hasher.combine(daysSinceLastExposure)
+		hasher.combine(validityDuration)
+	}
+
+	static func == (lhs: HomeHighRiskCellConfigurator, rhs: HomeHighRiskCellConfigurator) -> Bool {
```

Comment:
imo you should at least leave a comment why for the equality check you are using so many additional properties, but for the hasher, you are just using numberRiskContacts, daysSinceLastExposure and validityDuration.

Reply:
static func can't be overridden and I use all properties here, because 2 objects are equal when all them properties are equal ... hash method uses the same amount of properties, just some of them are calculated in the parent class